layouts/partials/components/edit-url.html

```diff
@@ -0,0 +1,21 @@
+{{- $editThisPage := (T "editThisPage") | default "Edit this page"}}
+{{- $linkClass := "mb-4 block text-xs text-gray-500 ltr:text-right rtl:text-left dark:text-gray-400" -}}
+{{- if and (site.Params.editURL.enable) (eq site.Params.editURL.location .source) -}}
+  {{- $editURL := site.Params.editURL.base | default "" -}}
+  {{- with .Params.editURL -}}
```

Comment:
the `.` here will not be able to access the page object. we should pass it down from where it's called. you can refer to other partials for how this can be achieved.

Reply:
as a result, the edit url will not be constructed correctly. for example: https://deploy-preview-290--hugo-hextra.netlify.app/docs/